utils/logger.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef ACCELERATOR_LOGGER_H_
+#define ACCELERATOR_LOGGER_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "utils/logger_helper.h"
+
+static inline bool ta_logger_init() {
+#ifdef NDEBUG
```

Comment:
It can be rewritten as following:
```c
#ifdef NDEBUG
  level = LOGGER_INFO;
#else
  level = LOGGER_DEBUG;
#endif
  if (logger_helper_init(level) != RC_OK) {
    return false;
```